src/types/index.ts

```diff
@@ -78,7 +78,14 @@ export namespace EnvDisplay {
  * | foo  | https://example.com
  */
 export namespace EnvList {
+  export enum EnvType {
```

Comment:
What do you think about not specifying a type for `type`? Instead we let plugin owners return whatever they want

I realize that we loose some type safety there but I think it makes a little more sense with our federated plugin model

Reply:
I am ok with simple strings. One thing to mention though is environment ordering. My plan was to use the enum as the key to order on, with and order of `Salesforce Orgs`, `Scratch Orgs` and then `Compute Environments`. With simple strings order would be the natural order.